### DISCUSSION TITLE
Can the new version of bolts be released synchronously with the lightning?

### DISCUSSION THREAD

**Discussion by zhutmost** [PyTorchLightning/pytorch-lightning-bolts#1003] [General]
Thanks a lot for the hard work of developers. I am using Lightning as well as bolts in my multiple research projects. 
Recently, lightning released v2.0.x version, and the bolts still depends on an older pytorch_lightning version (1.8?). 
The bolts project seems to be updated less frequently now, and its latest version is released at Dec 2022. 
So I cannot bump lightning version in my projects to the latest one.

Can we have a a roadmap of bolts?
It would be very helpful if the developers keep the bolts version upgrade synchronously with lightning.
I've waited a week or two since I completed upgrading the lightning code in my projects, and it looks like the bolts still don't have a release estimate. :(

Or just can we add bolts to the compatibility matrix [here](https://lightning.ai/docs/pytorch/stable/versioning.html#compatibility-matrix) .

Thank you~

**Comment by dwahdany** [comment]
You can build bolts from source (`pip install git+https://github.com/PytorchLightning/lightning-bolts.git@master --upgrade` or referencing a certain commit `pip install git+https://github.com/PytorchLightning/lightning-bolts.git@3b668ba515e6d9361137983c4f0c86d6c27f9398 --upgrade`) to use it with the latest pl (as of today). But that's just a workaround; I agree the versioning is unfortunate. It's been broken since at least January (#962)

**Reply by zhutmost** [reply]
Two week ago, even the master branch was broken and incompatible with pl 2.0. :(

**Reply by dwahdany** [reply]
Sorry, I just checked my environment and it seems the version I'm using (latest on conda) is 1.9.3. Maybe 2.0 is still broken, but compiling from master fixes at least #962

**Reply by zhutmost** [reply]
I have a look at the pull request page just now. And the version bumping is still incompleted and ongoing. #994 
Maybe I need to downgrade to version 1.9.X.
